.packit.yaml

```diff
@@ -110,6 +110,26 @@ jobs:
             context:
               revdeps: "yes"
 
+  - job: tests
+    identifier: cockpit-revdeps
+    trigger: pull_request
+    packages: [podman-centos]
+    notifications:
+      failure_comment:
+        message: "Cockpit tests failed for commit {commit_sha}. @martinpitt, @jelly, @mvollmer please check."
+    targets:
+      - centos-stream-10-x86_64
+    tf_extra_params:
+      environments:
+        - artifacts:
+          - type: repository-file
+            id: https://copr.fedorainfracloud.org/coprs/g/cockpit/main-builds/repo/fedora-$releasever/group_cockpit-main-builds-fedora-$releasever.repo
```

Comment:
fedora-releasever. It should be centos-stream-releasever

Reply:
my bad for not seeing that earlier.

Reply:
Exactly, sorry for the noise. Fixed now.